src/app/filename_formatter.cpp

```diff
@@ -112,14 +113,14 @@ std::string filename_formatter(
   base::replace_string(output, "{layer}", info.layerName());
   base::replace_string(output, "{group}", info.groupName());
   base::replace_string(output, "{slice}", info.sliceName());
-  base::replace_string(output, "{duration}", std::to_string(info.duration()));
 
   if (replaceFrame) {
     base::replace_string(output, "{tag}", info.innerTagName());
     base::replace_string(output, "{innertag}", info.innerTagName());
     base::replace_string(output, "{outertag}", info.outerTagName());
     replace_frame("{frame", info.frame(), output);
     replace_frame("{tagframe", info.tagFrame(), output);
+    replace_frame("{duration", info.duration(), output);
```

Comment:
Sorry didn't see it correctly, please don't use replace_frame for duration (it has a specific logic for frame numbers):
```suggestion
    base::replace_string(output, "{duration}", std::to_string(info.duration()));
```

Reply:
Ok! Thank you.